examples/pytorch/nlp/huggingface_models/language-modeling/pruning/eager/requirements.txt

```diff
@@ -6,4 +6,9 @@ torch
 tqdm
 optimum
 einops
-
+intel_extension_for_transformers
+tiktoken
+transformers_stream_generator
+peft
```

Comment:
Why do we require Perf and Deepspeed? Perhaps some other libraries aren't needed either